frontend/src/core/resource/components/ResourceMenu.js

```diff
@@ -117,7 +117,7 @@ export class ResourceMenuBase extends React.Component<Props, State> {
 
         let resourceName = parameters.resource.split('/').slice(-1)[0];
         if (parameters.resource === 'all-resources') {
-            resourceName = 'All Resources';
+            resourceName = <Localized id= 'resource-ResourceMenu--all-resources'>All Resources</Localized>;
```

Comment:
Nit: space.

Reply:
Should I also fix the use of Localized tag here? @mathjazz

Reply:
What do you mean by fix?